extensions/blocks/gif/editor.scss

```diff
@@ -27,17 +30,16 @@
 		flex-direction: row;
 		flex-wrap: wrap;
 		justify-content: center;
-		margin: 0 auto;
 		max-width: 400px;
 		width: 100%;
 		z-index: 1;
+		.components-text-control__input {
+			height: 36px;
```

Comment:
This shouldn't be needed, but the GIF block is doing bad things with the placeholder and label, so I'll clean that up...

Reply:
Awesome. Thank you!